.github/workflows/test.yml

```diff
@@ -11,14 +11,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-12, ubuntu-22.04, ubuntu-20.04]
+        os: [macos-14, ubuntu-22.04, ubuntu-24.04]
```

Comment:
Ubuntu 20.04 is still supported upstream and by GitHub actions so my preference is to keep testing against it.